tools/taskcluster/execute-firebase-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script installs google cloud sdk, logs into google firebase, executes tests,
+# and collects test artifacts into the test_artifacts folder
+
+# If a command fails then do not proceed and fail this script too.
+set -ex
+#########################
+# The command line help #
+#########################
+display_help() {
+    echo "Usage: $0 Build_Variant Apk_Name Device_Config [Device_Config...]"
+    echo
+    echo "Do not put .apk prefix in Apk_Name"
+    echo "Device_Config should be in landscape mode (e.g. model=taimen,version=26,orientation=landscape)"
+    echo
+}
+
+# Basic parameter check
+if [[ $# -lt 3 ]]; then
+    echo "Your command line contains $# arguments"
+    display_help
+    exit 1
```

Comment:
This error message would be pretty confusing out of context.  Could we print how many it should have, and why?  Maybe even include "See execute-firebase-test.sh for details"

Reply:
Actually it calls display_help which prints out the usage - I just added example use case to be more clear.